packages/file_selector/file_selector/.metadata

```diff
@@ -0,0 +1,30 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled.
+
+version:
+  revision: 4f9d92fbbdf072a70a70d2179a9f87392b94104c
+  channel: stable
+
+project_type: app
```

Comment:
This isn't the correct metadata. Please remove this file entirely; we've not been adding this retroactively since it would have misleading data about creation version.